automated install/basic-install.sh

```diff
@@ -378,6 +390,11 @@ setStaticIPv4() {
   local IFCFG_FILE
   local IPADDR
   local CIDR
+
+  if [[ ${usedhcpcd5} == true ]]; then
```

Comment:
This variable seems to be true on all Debian-like distros whatever the user decides. I see where this is getting, but I find the variable name misleading (`usedhcpcd5` can be true, although it will never be installed)